x11-misc/xkbd/xkbd-0.8.16-r1.ebuild

```diff
@@ -27,13 +26,10 @@ DEPEND="${RDEPEND}
 	x11-proto/xproto
 	x11-proto/xextproto"
 
-src_prepare() {
-	# 2008-03-23 gi1242: Fix handling of -geometry argument
-	epatch "${FILESDIR}"/${P}-fix-geometry.patch
-	# 2008-03-23 gi1242: Increase default repeat delay
-	epatch "${FILESDIR}"/${PN}-0.8.15-increase-delay.patch
-	epatch "${FILESDIR}"/${PN}-0.8.15-fix-keysyms-search.patch
-}
+PATCHES=( "${FILESDIR}"/${P}-fix-geometry.patch
+	"${FILESDIR}"/${P}-desktop.patch
+	"${FILESDIR}"/${PN}-0.8.15-increase-delay.patch
+	"${FILESDIR}"/${PN}-0.8.15-fix-keysyms-search.patch )
```

Comment:
In general, can you prefer:
```
PATCHES=(
	"${FILESDIR}"/${P}-fix-geometry.patch
	"${FILESDIR}"/${P}-desktop.patch
	"${FILESDIR}"/${PN}-0.8.15-increase-delay.patch
	"${FILESDIR}"/${PN}-0.8.15-fix-keysyms-search.patch
)
```
if there is **more than one** patch, as having them all at the same column makes things slightly easier to read